src/Data/Acid/Log.hs

```diff
@@ -268,8 +272,8 @@ archiveFileLog :: FileLog object -> EntryId -> IO ()
 archiveFileLog fLog entryId = do
   logFiles <- findLogFiles (logIdentifier fLog)
   let sorted = sort logFiles
-      relevant = filterLogFiles Nothing (Just entryId) sorted \\
-                 filterLogFiles (Just entryId) (Just (entryId+1))  sorted
+      relevant = filterLogFiles Nothing (Just entryId) sorted
```

Comment:
Is this just a stylistic choice or does it actually fix a compilation error? Not a complaint, just curious.

Reply:
Yes, I was surprised myself that CPP could not handle a line ending in `\\`, neither on GHC 7.8 nor the latest GHC 9.2.
Moving it away from the end of the line fixed the problem.
(Note that I added LANGUAGE CPP to this file to get a conditional import.)

Reply:
Turns out that I reported this GHC bug 5 year ago already:
- https://gitlab.haskell.org/ghc/ghc/-/issues/12391